<ui version="4.0">
 <class>TwcDialog</class>
 <widget class="QDialog" name="TwcDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Расчет суммарного водного течения</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_type">
     <property name="text">
      <string>Тип течения</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cmb_type"/>
   </item>
   <item>
    <widget class="QLabel" name="label_speed">
     <property name="text">
      <string>Скорость (узлы)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="spin_speed"/>
   </item>
   <item>
    <widget class="QPushButton" name="btn_cancel">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_save">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>